### PR TITLE
ArcticDB 3.0.0

### PR DESCRIPTION
**Pull request by poodlewars**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+{% set version = "3.0.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 149660905f2a1169a8bfeeb0f58e35e496747589a7d2147252181c4e826b3683
+  sha256: 592e9a66f2b991de095f83c3fca7cf9934d44315c38528abfd8b3c7a45381fac
```
Comment (poodlewars) [inline recipe/meta.yaml:10]: Force pushed new tag - will need to update the sha later

Comment (poodlewars) [reply recipe/meta.yaml:10]: I also probably need to include the entrypoint somehow... https://github.com/man-group/ArcticDB/blob/master/setup.cfg#L132

Comment (jjerphan) [reply recipe/meta.yaml:10]: For entry-points, according to [the documentation of `conda-build`](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#python-entry-points) (which used to build conda-forge feedstocks' recipe), reusing the same specification should be sufficient.

**Comment by jjerphan** [comment]
@poodlewars: can we close this PR in preference of #53?

**Pull request closed without merging**